exercises/patterns/printPlus/printPlus.js

```diff
@@ -39,8 +39,15 @@ function printPlus(height) {
 
   for (let i = 0; i < height; i++) {
     // This is your job. :)
-
-    helpers.printNewLine();
+    if(i === Math.floor(height/ 2)){
+      helpers.printCountTimes('#', height)
+      helpers.printNewLine();
+    }else{
+      let numSpaces = Math.floor(height / 2)
```

Comment:
You can move this line to line 39, just outside the for loop, because `numSpaces` has to be calculated just once, not at each loop.